src/rulesets/oas/ruleset.json

```diff
@@ -321,7 +321,7 @@
     "operation-tags": {
       "summary": "Operation should have non-empty `tags` array.",
       "type": "style",
-      "given": "$..paths.*[?( name() === 'get' || name() === 'put' || name() === 'post' || name() === 'delete' || name() === 'options' || name() === 'head' || name() === 'patch' || name() === 'trace' )]",
+      "given": "$..paths.*[?( @property === 'get' || @property === 'put' || @property === 'post' || @property === 'delete' || @property === 'options' || @property === 'head' || @property === 'patch' || @property === 'trace' )]",
```

Comment:
This is a breaking change for anyone who has written their own styleguide, right?

Reply:
That's right.